apps/preview/next/pages/showcases/Search/SearchWithButton.tsx

```diff
@@ -130,7 +130,7 @@ export default function SearchWithIcon() {
           value={searchValue}
           onChange={handleChange}
           onFocus={open}
-          wrapperClassName="w-full ring-0 active:ring-0 hover:ring-0 focus-within:ring-0 border-y border-l border-neutral-200 rounded-r-none hover:border-primary-800 active:border-primary-700 active:border-y-2 active:border-l-2 focus-within:border-y-2 focus-within:border-l-2 focus-within:border-primary-700"
+          wrapperClassName="w-full !ring-0 active:!ring-0 hover:!ring-0 focus-within:!ring-0 border-y border-l border-neutral-200 rounded-r-none hover:border-primary-800 active:border-primary-700 active:border-y-2 active:border-l-2 focus-within:border-y-2 focus-within:border-l-2 focus-within:border-primary-700"
```

Comment:
You've fully overwritten default behaviour. 🤔 Does it me it's a special case or the default behaviour is wrong?

Reply:
its a special case - we cant have "border" on the right due to the button. And ring always applies to all borders, you can't remove "part" of it.

Reply:
Alright, so it's fine to overwrite the default behaviour. 👍🏼